lib/matplotlib/lines.py

```diff
@@ -553,26 +553,26 @@ def set_markevery(self, every):
 (float, float) or list[bool]
             Which markers to plot.
 
-            - every=None, every point will be plotted.
-            - every=N, every N-th marker will be plotted starting with
+            - ``every=None``: every point will be plotted.
+            - ``every=N``: every N-th marker will be plotted starting with
               marker 0.
-            - every=(start, N), every N-th marker, starting at point
-              start, will be plotted.
-            - every=slice(start, end, N), every N-th marker, starting at
-              point start, up to but not including point end, will be plotted.
-            - every=[i, j, m, n], only markers at points i, j, m, and n
-              will be plotted.
-            - every=[True, False, True], positions that are True will be
+            - ``every=(start, N)``: every N-th marker, starting at index
+              *start*, will be plotted.
+            - ``every=slice(start, end, N)``: every N-th marker, starting at
+              index *start*, up to but not including index *end*, will be
               plotted.
-            - every=0.1, (i.e. a float) then markers will be spaced at
+            - ``every=[i, j, m, n]``: only markers at with the given indices
```

Comment:
this can be any list, right? 

```suggestion
            - ``every=[i, j, m, n, ...]``: only markers at with the given indices
```